server/routes/nl/helpers.py

```diff
@@ -404,10 +404,9 @@ def abort(error_message: str,
           blocked: bool = False,
           test: str = '',
           client: str = '') -> Dict:
-  query = str(escape(shared_utils.remove_punctuations(original_query)))
-  escaped_context_history = []
-  for ch in context_history:
-    escaped_context_history.append(escape(ch))
+  query = shared_utils.escape_strings(
```

Comment:
Why do we need escape_strings here?

Reply:
This functionality didn't change, we were always escaping this string, I'm just using a shared util for doing this instead